src/renderer/notification.ts

```diff
@@ -300,6 +300,21 @@ class Notification extends NotificationHandler {
         this.inactiveWindows = [];
     }
 
+    /**
+     * Brings all the notification to the top
+     * issue: ELECTRON-1382
+     */
+    public moveNotificationToTop(): void {
+        const notificationWindows = this.activeNotifications as ICustomBrowserWindow[];
+        notificationWindows
+            .filter((browserWindow) => typeof browserWindow.notificationData === 'object' && browserWindow.isVisible())
+            .forEach((browserWindow) => {
+                if (browserWindow && windowExists(browserWindow) && browserWindow.isVisible()) {
```

Comment:
browserWindow.isVisible() not required here.

Reply:
`isVisible` is required here because the notification window can be hidden any movement while in the loop